app/services/hearings_creator.rb

```diff
@@ -20,14 +20,24 @@ def call
 private
 
   def push_prosecution_cases
-    hearing[:prosecutionCases]&.each do |prosecution_case|
-      prosecution_case[:defendants].each do |defendant|
-        next if defendant[:offences].map { |offence| offence.dig(:laaApplnReference, :applicationReference)&.start_with?("A", "Z") }.any?
-
-        push_to_sqs(shared_time: shared_time,
-                    case_urn: prosecution_case[:prosecutionCaseIdentifier][:caseURN],
-                    defendant: defendant,
-                    appeal_data: nil)
+    hearing[:prosecutionCases]&.each do |prosecution_case_data|
+      prosecution_case_data[:defendants].each do |defendant_data|
+        next if defendant_data[:offences].any? { |offence| offence.dig(:laaApplnReference, :applicationReference)&.start_with?("A", "Z") }
+
+        laa_reference = LaaReference.find_by!(defendant_id: defendant_data[:id], linked: true)
+        next if laa_reference.blank?
```

Comment:
I don't think we need this. It wasn't in the original implementation, and we now know that all incoming cases are linked.

Reply:
Quite right. Thank you